### PR TITLE
Add-EntraScopedRoleMembership

### PR DESCRIPTION
**Pull request by v-uansari**

### CODE REVIEW

File: module/docs/entra-powershell-v1.0/Microsoft.Graph.Entra/Add-EntraScopedRoleMembership.md [hunk 1]
```diff
@@ -46,22 +47,31 @@ Connect-Entra -Scopes 'RoleManagement.ReadWrite.Directory'
...
 The example shows how to add a user to the specified role within the specified administrative unit.
 
+- `-ObjectId` Paramater specifies the ID of an administrative unit.
+- `-RoleObjectId` Paramater specifies the ID of a directory role.
+- `-RoleMemberInfo` Paramater specifies a RoleMemberInfo object.
+
 ## Parameters
 
-### -AdministrativeUnitId
+### -AdministrativeUnitObjectId
```
Comment (SteveMutungi254) [inline module/docs/entra-powershell-v1.0/Microsoft.Graph.Entra/Add-EntraScopedRoleMembership.md:74]: This should be removed. ObjectId is the right param.

Comment (snehalkotwal) [reply module/docs/entra-powershell-v1.0/Microsoft.Graph.Entra/Add-EntraScopedRoleMembership.md]: Changes done

**Review by KenitoInc — DISMISSED** [review]

**Review by SteveMutungi254 — APPROVED** [review]
LGTM.

**Review by SteveMutungi254 — APPROVED** [review]
LGTM.

**Pull request merged**